app/models/project_order.rb

```diff
@@ -0,0 +1,4 @@
+class ProjectOrder < ActiveRecord::Base
+	belongs_to :project 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.